worker/src/data.ts

```diff
@@ -4,6 +4,21 @@ export const KV_PREFIX_HISTORY = "history";
 export const KV_PREFIX_UUID = "uuid";
 export const KV_MAX_PROCESS_ENTRIES = 850;
 
+export interface Metadata {
```

Comment:
Since metadata is limited, can we make all keys single character? You can store the key in a constant to make the code readable `metadata[KEY_CREATED]` with `KEY_CREATED = 'c'`

Reply:
Sure, but why make it more complex? the payload fits?

Reply:
Because we don't know what we want/need to store tomorrow. Better have a solid foundation.